AptSpringAPI/src/main/java/com/salesforce/aptspring/Verified.java

```diff
@@ -32,20 +32,21 @@
 import java.lang.annotation.Target;
 
 /**
- * This marker annotation may only be used on Spring @Configuration and @Component classes.
+ * This marker annotation may only be used on Spring @Bean LITE and @Component classes.
  * Combined with the AptSpringProcessor annotation processor to prevent errors from creeping in to your spring graph!
  * <p>
  * The APT parser will generate errors if any of the following restrictions are violated:
  * </p>
  * <p>
- * Restrictions on @Configuration class definitions are:
+ * Restrictions on @Bean Lite mode class definitions are:
  * </p>
  * <ul>
- *  <li>All methods on an @Configuration class are @Bean methods.</li>
- *  <li>The @Configuration class has a public no-parameter constructor.</li>
- *  <li>The @Configuration class must be the top level class in a the java file.</li>
- *  <li>No uses of @ComponentScan on @Configuration classes, instead use @Import</li>
- *  <li>All fields on the @Configuration annotation must be "private static final {Type} {name} = {LITERAL_VALUE}";</li>
+ *  <li>All methods on the class are @Bean methods.</li>
+ *  <li>The class has a public no-parameter constructor.</li>
+ *  <li>The class must be the top level class in a the java file.</li>
+ *  <li>No uses of @ComponentScan on the classes, instead use @Import</li>
```

Comment:
@rexhoffman this comment is more related to a @Component not @Bean Lite mode right?

Reply:
Yes, this entire change list is forcing BeanLite mode when using @Bean definitions.  It is a no-op for standard (non @Bean method containing) @Components.

Reply:
Ok so we are saying only use @Import and @Component.  If @Bean is needed we can do it from within a @Component class right?  Can I use @Bean in a POJO and then just import via @Import  that POJO to register the lite bean with Spring?